source/imaer-shared/src/main/java/nl/overheid/aerius/shared/emissions/ColdStartEmissionsCalculator.java

```diff
@@ -41,7 +41,7 @@
  * 1000: divide by 1000 to convert gram to kilogram
  * </pre>
  */
-class ColdStartEmissionsCalculator extends BaseRoadEmissionsCalculator<ColdStartEmissionSource> {
+public class ColdStartEmissionsCalculator extends BaseRoadEmissionsCalculator<ColdStartEmissionSource> {
```

Comment:
Not entirely sure why this was made public?

Reply:
Looks like a remnant of an earlier development phase. It's not needed. So I've updated pr and reverted.